src/SFA.DAS.EAS.Web/Controllers/EmployerAccountController.cs

```diff
@@ -25,7 +25,7 @@ public ActionResult Gateway()
         [Route("gatewayResponse")]
         public ActionResult GateWayResponse()
         {
-            return Redirect(Url.EmployerAccountsAction($"gatewayResponse?{Request.QueryString}"));
+            return Redirect(Url.EmployerAccountsAction($"gatewayResponse{Request?.Url?.Query}"));

```

Comment:
Can Url and Query actually ever be null?

Reply:
its just the Query from the looks of it. i'll update the code now

Reply:
Url can be null as its a public variable that can be set and given its a complex type I'm guessing they have to cover for it being null. Not sure if it could ever be null be default however with regards to this action.